pbiviz.json

```diff
@@ -2,14 +2,14 @@
   "visual": {
     "name": "PowerBI-visuals-forcasting-exp",
     "displayName": "Forecasting",
-    "guid": "PBI_CV_8EDDC07B_EE79_4418_A84C_D73897C0E21F",
+    "guid": "PBI_CV_8EDDC07B_EE79_4418_A84C_D73897C0E21F_HTML",
     "visualClassName": "Visual",
-    "version": "1.0.2",
-    "description": "<span>Time series forecasting is the use of a model to predict future values based on previously observed values. Current visual implements well known exponential smoothing method for the forecasting. The prediction is based on trend and seasonality modeling. You can control the algorithm parameters and the visual attributes to suit your needs.<br/><br/><span style='font-style:italic'>Service prerequisites:</span> R-powered custom visual is used in service seamlessly<br/><br /><span style='font-style:italic'>Desktop prerequisites:</span> To run R scripts in Power BI Desktop, you must separately install R on your local computer.<br />You can download and install R for free from the <a href='https://mran.revolutionanalytics.com/download/'>Revolution Open download page</a> or the <a href='https://cran.r-project.org/bin/windows/base/'>CRAN Repository</a><br /><br /> <span style='font-style:italic'> R package dependencies(auto-installed): </span> graphics, scales, forecast, zoo <br /><br /> <span style='font-style:italic'> Supports R versions: </span> R 3.3.1, R 3.3.0, MRO 3.3.1, MRO 3.3.0, MRO 3.2.2  <br /></span>",
+    "version": "1.0.3",
+    "description": "<span>Time series forecasting is the use of a model to predict future values based on previously observed values. Current visual implements well known exponential smoothing method for the forecasting. The prediction is based on trend and seasonality modeling. You can control the algorithm parameters and the visual attributes to suit your needs.<br/><br/><span style='font-style:italic'>Service prerequisites:</span> R-powered custom visual is used in service seamlessly<br/><br /><span style='font-style:italic'>Desktop prerequisites:</span> To run R scripts in Power BI Desktop, you must separately install R on your local computer.<br />You can download and install R for free from the <a href='https://mran.revolutionanalytics.com/download/'>Revolution Open download page</a> or the <a href='https://cran.r-project.org/bin/windows/base/'>CRAN Repository</a><br /><br /> <span style='font-style:italic'> R package dependencies(auto-installed): </span> graphics, scales, forecast, zoo, ggplot2, htmlWidgets, XML, plotly  <br /><br /> <span style='font-style:italic'> Supports R versions: </span> R 3.3.1, R 3.3.0, MRO 3.3.1, MRO 3.3.0, MRO 3.2.2  <br /></span>",
     "supportUrl": "http://community.powerbi.com/",
-    "gitHubUrl": "https://github.com/microsoft/PowerBI-visuals-forcasting-exp"
+    "gitHubUrl": "https://github.com/microsoft/PowerBI-visuals-forcasting-exp/tree/HTML-based"
```

Comment:
I suppose that we should leave link as is. This pull request assumes that we will merge into master without creating a new branch.